server/data/migrations/20190823091110-create-tables.js

```diff
@@ -0,0 +1,358 @@
+  up: (queryInterface, Sequelize) => queryInterface.sequelize.query('CREATE EXTENSION IF NOT EXISTS pgcrypto;').then(() => queryInterface.sequelize.transaction(transaction => Promise.all([
+    queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.literal('gen_random_uuid()')
+      },
+      username: {
+        type: Sequelize.STRING,
+        unique: true
+      },
+      email: {
+        allowNull: false,
+        type: Sequelize.STRING,
+        unique: true
+      },
+      password: {
+        type: Sequelize.STRING,
+        unique: false
+      },
+      name: {
+        type: Sequelize.STRING,
+        unique: false
+      },
+      bio: {
+        type: Sequelize.STRING,
+        unique: false
+      },
+      url: {
+        type: Sequelize.STRING,
+        unique: false
+      },
+      company: {
+        type: Sequelize.STRING,
+        unique: false
+      },
+      location: {
+        type: Sequelize.STRING,
+        unique: false
+      },
+      imgUrl: {
+        type: Sequelize.STRING,
+        unique: false
+      },
+      type: {
+        allowNull: false,
+        type: Sequelize.TEXT,
+        unique: false,
+        defaultValue: 'USER'
+      },
+      fake: {
+        allowNull: false,
+        type: Sequelize.BOOLEAN,
+        unique: false,
+        defaultValue: false
+      },
+      createdAt: Sequelize.DATE,
+      updatedAt: Sequelize.DATE,
+      deletedAt: Sequelize.DATE
+    }, { transaction }),
+    queryInterface.createTable('sshKeys', {
+      id: {
+        allowNull: false,
+        autoIncrement: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.literal('gen_random_uuid()')
+      },
+      value: {
+        allowNull: false,
+        type: Sequelize.TEXT
+      },
+      title: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      fingerprint: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      userId: {
+        allowNull: false,
+        type: Sequelize.UUID,
+        references: {
+          model: 'users',
+          key: 'id'
+        }
+      },
+      createdAt: Sequelize.DATE,
+      updatedAt: Sequelize.DATE,
+      deletedAt: Sequelize.DATE
+    }, { transaction }),
+    queryInterface.createTable('repositories', {
+      id: {
+        allowNull: false,
+        autoIncrement: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.literal('gen_random_uuid()')
+      },
+      name: {
+        allowNull: false,
+        type: Sequelize.TEXT
+      },
+      userId: {
+        allowNull: false,
+        type: Sequelize.UUID,
+        references: {
+          model: 'users',
+          key: 'id'
+        }
+      },
+      description: {
+        type: Sequelize.TEXT,
+        unique: false
```

Comment:
doesn't unique equal to false by default?

Reply:
fixed